src/ILP/default-ilp-client.ts

```diff
@@ -37,45 +37,42 @@ class DefaultIlpClient implements IlpClientDecorator {
   public constructor(private readonly networkClient: IlpNetworkClient) {}
 
   /**
-   * Retrieve the balance for the given address.
+   * Retrieve the balance for the given accountId.
```

Comment:
nit: Id to ID to be consistent with the rest of the SDK code. 

I'm guessing by the amount of time I make this comment that the ILP team favors "Id" so maybe we can change this at some point in the future.